packages/integration-tests/tests/me.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ArbitraryObject, User } from '@logto/schemas';
+
+import api, { authedAdminApi } from '@/api';
+import { generatePassword, generateUsername } from '@/utils';
+
+const createUser = async () => {
+  const username = generateUsername();
+
+  return authedAdminApi
+    .post('users', {
+      json: {
+        username,
+        password: generatePassword(),
```

Comment:
better inline both `username` and `password`?

Reply:
I prefer to set the `username` and `name` with the same value, so the username is not inline.